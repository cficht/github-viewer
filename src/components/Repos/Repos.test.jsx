import React from 'react';
import { shallow } from 'enzyme';
import Repos from './Repos';

describe('Repos component', () => {
  it('renders Repos', () => {
    const repos = [
      {
        id: 233724660,
        name: 'about-me',
        html_url: 'https://github.com/cficht/about-me'
      },
      {
        id: 250081461,
        name: 'book-application',
        html_url: 'https://github.com/cficht/book-application'
      }
    ];
    const wrapper = shallow(<Repos repos={repos}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
