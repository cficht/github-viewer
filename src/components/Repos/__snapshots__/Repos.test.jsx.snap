// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repos component renders Repos 1`] = `
<ul>
  <Repo
    html_url="https://github.com/cficht/about-me"
    id={233724660}
    key="233724660"
    name="about-me"
  />
  <Repo
    html_url="https://github.com/cficht/book-application"
    id={250081461}
    key="250081461"
    name="book-application"
  />
</ul>
`;
